tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2995,11 +2995,10 @@ def rgb_to_yuv(images):
     images: tensor with the same shape as `images`.
     
   Usage Example:
-  ```python
-  >> import tensorflow as tf
-  >> x = tf.random.normal(shape=(256, 256, 3))
-  >> tf.image.rgb_to_yuv(x)
-  ```
+  
+  >>> x = tf.random.normal(shape=(256, 256, 3))
+  >>> tf.image.rgb_to_yuv(x)
+  <tf.Tensor 'Tensordot_5:0' shape=(256, 256, 3) dtype=float32>
```

Comment:
This was built using tf 1.x not tf2; in tf2 tensors have values

Reply:
Yap that's true Let me change it

Reply:
The output in tf2 has numpy arrays should include it in the all i should add (```) before  and after the the code?